### ISSUE TITLE
setConfig keeps adding elements

### ISSUE THREAD

**Issue by thomasloven** [custom-cards/bar-card#23]
Hi.

Every time `setConfig` is run, a new ha-card is added to the shadowRoot, but it's never removed.

`setConfig` can be run several times during the lifetime of the card, e.g. if the user uses it with an `entity-filter` card.

An easy fix is to add the following somewhere before `this.shadowRoot.appendchild(cardContainer)` inside `setConfig()`:

```javascript
while(this.shadowRoot.lastChild)
  this.shadowRoot.removeChild(this.shadowRoot.lastChild);
```

See /thomasloven/lovelace-auto-entities#31

**Comment by Gluwc** [comment]
Thanks for reporting this. I had this in the script initially, but removed it at some point. Probably because I had no idea what I was doing... and still don't really ;)

Fixed in [1.6.2](https://github.com/custom-cards/bar-card/releases/tag/1.6.2).

**Comment by thomasloven** [comment]
That makes two of us.
